pkg/plugins/pfs/pfs.go

```diff
@@ -49,7 +49,7 @@ type slotandname struct {
 var allslots []slotandname
 
 func init() {
-	var all []string
+	all := make([]string, 0, len(PermittedCUEImports()))
```

Comment:
Nit; Since it is a function call, I'd normally allocate the result into a variable and reuse it, tho it's not a great deal because it just returns a static and small slice of strings 🤷🏻